### ISSUE TITLE
Strange entries in data/tgst5.dat

### ISSUE THREAD

**Issue by IzzySoft** [pkumza/LibRadar#7]
Some entries in `data/tgst5.dat` seem to be syntactically incorrect. As far as I understood, syntax for the library name is `tt;name;url` (with `tt` standing for the type of the library). Now I found a couple of entries where the `url` is not an URL at all, e.g.

* `"ut;Tapit;Tapit NFC\u6280\u672f"`
* `"da;jnlua;Java Lua \u4e92\u8054\u5de5\u5177"`

Do I misinterprete something, and that's intended? Or should it be fixed? Please clarify and let me know how to deal with such entries. If I'm assuming right concerning the syntax, those entries should be fixed up to

* `"ut;Tapit NFC;"` (couldn't find a reference URL)
* `"da;Java Lua;"` (no reference URL here either)

**Comment by pkumza** [comment]
Yes, that's an error from the former version. Some introductions were written in Chinese.

**Comment by IzzySoft** [comment]
OK, thanks. So if you're not opposed, I'll go over those and fix them up with a separate PR – and use the commit message of that to auto-close this issue (`fixes #7`). Before starting on that, I'll re-check with this issue if you didn't want to stop me – or already have fixed that by another commit meanwhile, of course :)
